src/client/sandbox/code-instrumentation/properties/index.js

```diff
@@ -136,7 +136,12 @@ export default class PropertyAccessorsInstrumentation extends SandboxBase {
             action: {
                 condition: el => domUtils.isDomElement(el) && domProcessor.isUrlAttr(el, 'action'),
 
-                get: el => PropertyAccessorsInstrumentation._getUrlAttr(el, 'action'),
+                get: el => {
+                    if (domUtils.isDomElement(el.action))
```

Comment:
I propose to invert logic:
```js
if typeof el.action === 'string'
   return PropertyAccessorsInstrumentation._getUrlAttr(el, 'action');

return el.action;
```
It helps to prevent edge cases when form.action is not dom element.
And add test for this case